### PR TITLE
Add Contact List

### PR DESCRIPTION
**Pull request by Khikar-1997**

### CODE REVIEW

File: src/Screens/Components/ContactList/Avatar.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```
Comment (griarman) [inline src/Screens/Components/ContactList/Avatar.js]: please add case for users that hadn't images

File: src/Screens/Components/ContactList/Avatar.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import { Image, StyleSheet, View } from 'react-native';
+
+const Avatar = ({ contactList }) => {
+  return (
+    <View style={styles.imageContainer}>
+      <Image
+        source={{
+          uri: contactList.item.avatar_image,
```
Comment (griarman) [inline src/Screens/Components/ContactList/Avatar.js]: add question marks

File: src/Screens/Components/ContactList/Avatar.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        }}
+        style={styles.avatarImage}
+      />
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  imageContainer: {
+    backgroundColor: '#1B2124',
```
Comment (griarman) [inline src/Screens/Components/ContactList/Avatar.js]: please create COLORS file and add your colors there

File: src/Screens/Components/ContactList/ContactInfo.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Text, View, StyleSheet } from 'react-native';
+
+const ContactInfo = ({ contactList }) => {
```
Comment (griarman) [inline src/Screens/Components/ContactList/ContactInfo.js]: if in your component you don't write any logic, you don't need return

File: src/Screens/Components/ContactList/ContactsList.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { FlatList, StyleSheet, View } from 'react-native';
+import MOCK_DATA from '../../../data/MOCK_DATA.json';
+import Avatar from './Avatar';
+import ContactInfo from './ContactInfo';
+
+const ContactsList = () => {
+  return (
+    <View>
```
Comment (griarman) [inline src/Screens/Components/ContactList/ContactsList.js]: this is unnecessary View component

File: src/Screens/Components/ContactList/ContactsList.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+      <FlatList
+        data={MOCK_DATA}
+        renderItem={contactList => {
+          return (
+            <View style={styles.contactListContainer}>
+              <Avatar contactList={contactList} />
+              <ContactInfo contactList={contactList} />
+            </View>
+          );
```
Comment (griarman) [inline src/Screens/Components/ContactList/ContactsList.js]: you can create component for this part

**Review by griarman — CHANGES_REQUESTED** [review]

File: src/Components/ContactList/ContactInfo.js [hunk 7]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Text, View, StyleSheet } from 'react-native';
+
+import colors from '../../Constants/colors';
+
+const ContactInfo = ({ name, surname, phoneNumber }) => (
+  <View style={styles.contactInfoContainer}>
+    <Text style={styles.contactNameAndSurnameText}>
+      {surname ? `${name} ${surname}` : `${name}`}
```
Comment (griarman) [inline src/Components/ContactList/ContactInfo.js]: `${name} ${surname || ''}`

File: src/Screens/Contacts.screen.js [hunk 8]
```diff
@@ -1,14 +1,14 @@
...
+import ContactsList from '../Components/ContactList/ContactsList';
 
-const Contacts = ({ navigation }) => {
-  return (
-    <View style={styles.contactScreen}>
-      <Button title="Add" onPress={() => navigation.navigate(ADD_CONTACTS)} />
-    </View>
-  );
-};
+const Contacts = ({ navigation }) => (
```
Comment (griarman) [inline src/Screens/Contacts.screen.js]: you can use useNavigation hook

File: src/UIKit/Avatar/index.js [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+        source={{
+          uri: image,
+        }}
+        style={styles.avatarImage}
+      />
+    ) : (
+      <Text style={styles.avatarNameFirstLetter}>
+        {surname
+          ? `${name.charAt(0).concat(surname.charAt(0))}`
+          : `${name.charAt(0)}`}
```
Comment (griarman) [inline src/UIKit/Avatar/index.js]: please beautify this

**Review by griarman — CHANGES_REQUESTED** [review]

**Review by AnushavanGhulyan — APPROVED** [review]

**Pull request merged**